src/main/java/Bocchi.java

```diff
@@ -0,0 +1,61 @@
+import exceptions.BocchiException;
+import parser.Parser;
+import storage.BocchiLoader;
+import storage.BocchiSaver;
+import task.TaskList;
+import ui.Ui;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+/**
+ * Class for running the task manager
+ */
+public class Bocchi {
+    private final Ui ui;
+    private final Parser parser;
+
+    /**
+     * Initializes the program
+     */
+    public Bocchi() {
+        this.ui = new Ui();
+        this.parser = new Parser(this.ui);
+    }
+
+    /**
+     * Main method.
+     */
+    public static void main(String[] args) {
+        new Bocchi().run();
+    }
+
+    /**
+     * Runs the program
+     */
```

Comment:
Perhaps you can abstract this `run()` method further, e.g. make an `initialise()` method to instantiate your scanner and storage etc. That way `run()` will only contain logic relevant to the uptime of the program.